Xamarin.Forms.Platform.Android/VisualElementTracker.cs

```diff
@@ -199,13 +197,9 @@ void HandleViewAttachedToWindow()
 			UpdateClipToBounds();
 		}
 
-		void MaybeRequestLayout()
+		void MaybeRequestLayout(bool forceRequest)
 		{
-			var isInLayout = false;
-			if ((int)Build.VERSION.SdkInt >= 18)
```

Comment:
We support actively 19+, but we also don't want to break anything lower if we can help it.